docs/api-guide/serializers.md

```diff
@@ -774,6 +774,11 @@ Here's an example of how you might choose to implement multiple updates:
 
     class BookSerializer(serializers.Serializer):
         ...
+        id = serializers.IntegerField(
+            read_only=False,
+            required=False
+        )
```

Comment:
1. Presumably we *do* actually want it to be required.
2. The default for `read_only` is `False` anyway so we don't need to specify that.
3. Let's add a comment explaining why this is here.

Eg. How does this look?

    # Specify the id field explicitly so that we get a writable IntegerField,
    # rather than a read-only field, that primary keys are normally mapped to.
    id = serializers.IntegerField()